src/Private/Version.ps1

```diff
@@ -0,0 +1,127 @@
+<#
+.SYNOPSIS
+Retrieves the PowerShell module manifest object for the specified module.
+
+.DESCRIPTION
+This function constructs the path to a PowerShell module manifest file (.psd1) located in the parent directory of the script root. It then imports the module manifest file to access its properties and returns the manifest object. This can be useful for scripts that need to dynamically discover and utilize module metadata, such as version, dependencies, and exported functions.
+
+.PARAMETERS
+This function does not accept any parameters.
+
+.EXAMPLE
+$manifest = Get-ModuleManifest
+This example calls the `Get-ModuleManifest` function to retrieve the module manifest object and stores it in the variable `$manifest`.
+
+#>
+function Get-ModuleManifest {
+    # Construct the path to the module manifest (.psd1 file)
+    $moduleManifestPath = Join-Path -Path (Split-Path -Path $PSScriptRoot -Parent) -ChildPath 'Pode.psd1'
+
+    # Import the module manifest to access its properties
+    $moduleManifest = Import-PowerShellDataFile -Path $moduleManifestPath
+    return  $moduleManifest
+}
+
+
+<#
+.SYNOPSIS
+Gets the version of the Pode module.
+
+.DESCRIPTION
+The Get-PodeVersion function checks the version of the Pode module specified in the module manifest. If the module version is not a placeholder value ('$version$'), it returns the actual version prefixed with 'v.'. If the module version is the placeholder value, indicating the development branch, it returns '[develop branch]'.
+
+.PARAMETER None
+This function does not accept any parameters.
+
+.OUTPUTS
+System.String
+Returns a string indicating the version of the Pode module or '[develop branch]' if on a development version.
+
+.EXAMPLE
+PS> $moduleManifest = @{ ModuleVersion = '1.2.3' }
+PS> Get-PodeVersion
+
+Returns 'v.1.2.3'.
+
+.EXAMPLE
+PS> $moduleManifest = @{ ModuleVersion = '$version$' }
+PS> Get-PodeVersion
+
+Returns '[develop branch]'.
+
+.NOTES
+This function assumes that $moduleManifest is a hashtable representing the loaded module manifest, with a key of ModuleVersion.
+
+#>
+function Get-PodeVersion {
+    $moduleManifest = Get-ModuleManifest
+    if ($moduleManifest.ModuleVersion -ne '$version$') {
+        return "v.$($moduleManifest.ModuleVersion)"
+    }
+    else {
+        return '[develop branch]'
```

Comment:
It might be best to return just `[dev]`, as a person could be on a different branch and not the develop one :)

Reply:
fare